### PR TITLE
Improve docs for sugar3.activity.activityhandle

### PR DESCRIPTION
**Pull request by ndefilippis**

### CODE REVIEW

File: src/sugar3/activity/activityhandle.py [hunk 1]
```diff
@@ -15,47 +15,47 @@
...
-                   from the network
-        """
         self.activity_id = activity_id
         self.object_id = object_id
         self.uri = uri
         self.invited = invited
 
     def get_dict(self):
-        """Retrieve our settings as a dictionary"""
+       '''Returns activity settings as a dictionary in format {activity_id:XXXX, object_id:XXXX, uri:XXXX, invited:BOOL}'''
```
Comment (samdroid-apps) [inline src/sugar3/activity/activityhandle.py]: You've done something funny with the indent, as sugar crashes when this patch is applied:

```
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/saam/sugar-build/build/out/install/lib/python2.7/site-packages/jarabe/main.py", line 66, in <module>
    from jarabe.view import keyhandler
  File "/home/saam/sugar-build/build/out/install/lib/python2.7/site-packages/jarabe/view/keyhandler.py", line 33, in <module>
    from jarabe.journal import journalactivity
  File "/home/saam/sugar-build/build/out/install/lib/python2.7/site-packages/jarabe/journal/journalactivity.py", line 32, in <module>
    from sugar3.activity import activityfactory
  File "/home/saam/sugar-build/build/out/install/lib/python2.7/site-packages/sugar3/activity/activityfactory.py", line 30, in <module>
    from sugar3.activity.activityhandle import ActivityHandle
  File "/home/saam/sugar-build/build/out/install/lib/python2.7/site-packages/sugar3/activity/activityhandle.py", line 59
    result = {'activity_id': self.activity_id,
    ^
IndentationError: unexpected indent
```

Also, please keep lines shorter than 80 charecters.  This one looks too long.

**Comment by ndefilippis** [comment]
I was one space short. Should be fixed now

**Comment by quozl** [comment]
Reviewed.

**Comment by samdroid-apps** [comment]
@quozl in future, feel free to merge doc pull requests :)

**Pull request merged**